src/makefile

```diff
@@ -29,4 +29,4 @@ cryptosystem/cryptosystem.cc
 ############################################################
 
 clean:
-	rm -f *.o $(EXECUTABLE)
+	rm *.o $(EXECUTABLE)
```

Comment:
`rm -f` to silently remove non-existing files
- avoiding an error message in case `make clean` is invoked when some `.o` file has not been created